woocommerce/Settings_API/Setting.php

```diff
@@ -55,6 +55,256 @@ class Setting {
 	const TYPE_BOOLEAN = 'boolean';
 
 
+	/** @var string unique setting ID */
+	protected $id;
+
+	/** @var string setting type */
+	protected $type;
+
+	/** @var string setting name */
+	protected $name;
+
+	/** @var string setting description */
+	protected $description;
+
+	/** @var bool whether the setting holds an array of multiple values */
+	protected $is_multi = false;
+
+	/** @var array valid setting options */
+	protected $options = [];
+
+	/** @var int|float|string|bool|array setting default value */
+	protected $default;
+
+	/** @var int|float|string|bool|array setting current value */
+	protected $value;
+
+	/** @var Control control object */
+	protected $control;
+
+
+	/** Setter Methods ************************************************************************************************/
+
+
+	/**
+	 * Sets the setting ID.
+	 *
+	 * @since x.y.z
+	 *
+	 * @param string $id
+	 */
+	public function set_id( $id ) {
+		$this->id = $id;
```

Comment:
The setters and getters are missing an empty line at the beginning at the method.